ReactSkia/sdk/OSKLayout.h

```diff
@@ -0,0 +1,244 @@
+ /*
+ * Copyright (C) 1994-2022 OpenTV, Inc. and Nagravision S.A.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#ifndef OSK_LAYOUT_H
+#define OSK_LAYOUT_H
+
+#include "RNSKeyCodeMapping.h"
+
+using namespace OSK;
+
+/*
+Position Configuration mentioned in this file should be for 1280*720 screen.
+For other Screen Size, it will be scaled accordingly in SW w.r.t 1280*720.
+*/
+static SkSize baseScreenSize{1280,720};
+
+#define LOWER_TO_UPPER_CASE_OFFSET 32
+
+//Partition or Key Groups a Layout can have
+enum partitionID {
+    GROUP1,
+    GROUP2,
+    GROUP3,
+    GROUP4,
+    MAX_GROUP=GROUP4
+};
```

Comment:
Can we add comment for every group indicating what they mean. Also, If possible give meaningful name for GROUP instead of giving number suffix ?

Reply:
Enum as per it's name  here to define the max partitionID/ Group, OSk Layout can handle.

Each Group get it name/function/meaning, when Layout is declared. In Layout declaration,it was mentioned description of group & it's function

example :- Is this Fine?
keyPlacementConfig_t alphaNumericKBGroupConfig[ALPHA_NUMERIC_KB_PARTITION_COUNT]={
    //GROUP1-Toggle key
    {{20,60},{15,30},1.3,0.75,4},
    //GROUP2-Numeric key
    {{150,40},{15,15},1.25,1,1},
    //GROUP3-Alpha & SYmbol key
    {{320,40},{15,15},1.25,1,1},
    //GROUP4-Function key
    {{820,50},{15,15},1.5,1,1}
};